spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java

```diff
@@ -407,7 +407,7 @@ else if (property.getAnnotatedColumnItemType() != null) {
 
 		if (!valueSet) {
 			throw new SpannerDataException(String.format(
-					"Unsupported mapping for type: %s", propertyValue.getClass()));
```

Comment:
In principle, propertyType can be null. Not sure whether it would happen in practice.

Reply:
Every property has to have a type right (meaning it would be non-null)? Not sure. But this will solve our NPE here from `.getClass()` though.